### DISCUSSION TITLE
Unable to use lfrc config file on windows

### DISCUSSION THREAD

**Discussion by jcbsorensen** [gokcehan/lf#1343] [Q&A] [answered]
Hello :)

I am using lf on windows 10 with powershell. I need to add some extra configurations to my lf usage - especially some kind of delete function. I have created a setting file in "myuser/.config/lf/lfrc" with basic setup code from the github (lfrc.ps1.example). However, it seems LF does not use this config file. When "set shell powershell" is specified in the config file, LF does not give me access to the powershell shell. Likewise, when I create a simple mapping such as "map . hidden!", LF continues to regard "." as unmapped.

Is there some kind of basic setup of variables I need to setup before I can get full funcitonality of LF?

- Jacob

**Comment by ilyagr** [comment, marked as answer]
On Windows, these files need to be in a weird location. Take a look at https://pkg.go.dev/github.com/gokcehan/lf#hdr-Configuration.

**Reply by jcbsorensen** [reply]
Thank you so much :) that did the trick.

**Comment by cpbotha** [comment]
That link does not go to the right place anymore. Try his one: https://github.com/gokcehan/lf/blob/master/doc.md#configuration

Also, let me extract the actual answer to help future travellers:

```
OS       system-wide               user-specific
Unix     /etc/lf/lfrc              ~/.config/lf/lfrc
Windows  C:\ProgramData\lf\lfrc    C:\Users\<user>\AppData\Local\lf\lfrc
```
